engineyard/ey-cookbooks-stable-v5
# Remove =sys-apps/yarn-0.21.3 from cookbooks. Install =sys-apps/yarn-0.21.3-r1 from EYGL.

Ticket
-------------
https://tickets.engineyard.com/issue/CC-1132

Description of your patch
-------------
Remove =sys-apps/yarn-0.21.3 from cookbooks. Install =sys-apps/yarn-0.21.3-r1 from EYGL.

Recommended Release Notes
-------------
We've removed the installation of =sys-apps/yarn-0.21.3 from cookbooks. We now install =sys-apps/yarn-0.21.3-r1 from the operating system itself.

Estimated risk
-------------
=sys-apps/yarn-0.21.3-r1 should be a full replacement for =sys-apps/yarn-0.21.3 from cookbooks. However, yarn's program files are now kept in appropriate system locations instead of all under /opt.

Components involved
-------------
EYGL, cookbooks, chef, emerge (portage), yarn

Description of testing done
-------------
Ensured a cookbooks installed the expected version of sys-apps/yarn (=sys-apps/yarn-0.21.3-r1).

QA Instructions
-------------
Test in a from-scratch stack. =sys-apps/yarn-0.21.3-r1 should be the first and only version of sys-apps/yarn installed by the first chef run. Verify with 'qlop --list sys-apps/yarn'. I believe you may need to use a node.js app (as I have) to exercise this code path?
